frontend/src/helpers/time.helper.ts

```diff
@@ -21,11 +21,14 @@ export const timeFormat = (seconds: number) => {
 
   if (bufer >= SECONDS_IN_MINUTE) {
     const minutes = Math.floor(bufer / SECONDS_IN_MINUTE);
-    result += minutes.toString() + 'm ';
+    result += minutes.toString() + 'm';
     bufer = bufer - SECONDS_IN_HOUR * minutes;
   }
 
-  if (!result) result = '0m';
+  if (!result) {
+    if (bufer > 0) result = bufer + 's';
+    else result = '0m';
```

Comment:
Use ternary operator here. And do not use if/else wihout braces.

Reply:
result = condition ? true : false;